integration/kubernetes/helm-chart/alluxio/templates/master/statefulset.yaml

```diff
@@ -9,68 +9,61 @@
 # See the NOTICE file distributed with this work for information regarding copyright ownership.
 #
 
-{{ $root := . -}}
```

Comment:
Removed all the `$root` ref since there's no more range loop that changes the scope of variables.